schema.sql

```diff
@@ -0,0 +1,54 @@
+CREATE DATABASE CATALOG;
+
+CREATE TABLE  ITEM (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL
+);
+
+CREATE TABLE BOOK (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  publisher VARCHAR(50) NOT NULL,
+  cover_state VARCHAR(50) NOT NULL,
+  FOREIGN KEY (id) REFERENCES item(id) ON DELETE CASCADE
+);
+
+CREATE TABLE LABEL (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  title VARCHAR(50) NOT NULL,
+  color VARCHAR(50) NOT NULL,
+  book_id INT,
+  FOREIGN KEY (book_id) REFERENCES item(id) ON DELETE CASCADE
+);
+
+CREATE TABLE GAME (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  multiplayer VARCHAR(50) NOT NULL,
+  last_player_at DATE NOT NULL,
+  item_id INT,
+  FOREIGN KEY (item_id) REFERENCES item(id) ON DELETE CASCADE
+);
+
+CREATE TABLE AUTHOR (
+  id INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  first_name VARCHAR(50) NOT NULL,
+  last_name VARCHAR(50) NOT NULL,
+  author_id INT,
+  FOREIGN KEY (auothor_id) REFERENCES item(id) ON DELETE CASCADE
+);
+
+-- Create genres table
+CREATE TABLE genres (
+  id INT Generated ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(100),
+);
```

Comment:
- [ ] Good job with the schema and your entire code base in general 👏 .  However, I noticed some of your tables were not exactly implemented as it is on the UML diagram. you're missing the `item:array` properties. Please check the image below and effect changes to the affected tables 👍 

## View:
![image](https://github.com/AbdusSattar-70/Ruby-Group-Capstone/assets/50990925/ce7be627-097a-48fd-8c6c-6ac7b47efa80)

Reply:
Hello @Shedrack-Sunday thanks for your valuable suggestion please give me an example of 
how can we implement it on schema.sql file

Reply:
Hello @AbdusSattar-70 try this:
``` 
CREATE TABLE Genre (
  id INT,
  name VARCHAR(255),
  items TEXT[]
);
``` 

You can also make it a json

Reply:
@Shedrack-Sunday Thanks for your example suggestion  👍

Reply:
You're welcome